x-pack/plugins/security_solution/public/entity_analytics/api/api.ts

```diff
@@ -139,6 +141,24 @@ export const useEntityAnalyticsRoutes = () => {
       });
     };
 
+    const uploadAssetCriticalityFile = async (
+      fileContent: string,
+      fileName: string
+    ): Promise<AssetCriticalityCsvUploadResponse> => {
+      const file = new File([new Blob([fileContent])], fileName, { type: 'text/csv' });
+      const body = new FormData();
+      body.append('file', file);
+
+      return http.fetch<AssetCriticalityCsvUploadResponse>(ASSET_CRITICALITY_CSV_UPLOAD_URL, {
+        version: '1',
+        method: 'POST',
+        headers: {
+          'Content-Type': undefined, // Required by BE
```

Comment:
// Lets the browser set the appropriate content type

Someone else put it better elsewhere in Kibana :D